src/styles/index.css

```diff
@@ -0,0 +1,73 @@
+@import "./reset.css";
+@import "./stars.css";
+@import "./colors.css";
```

Comment:
Leave this here, and everythings else, what is down below, you should place to a separate file named as `card-block.css`